.gitignore

```diff
@@ -16,6 +16,8 @@ pom.xml.asc
 .hg/
 .idea
 beat-link-trigger.iml
-
+.github/scripts/**/*
+*.log
```

Comment:
How did you end up with log files at this level such that you wanted to exclude them?